### ISSUE TITLE
3D Camera Ease of Use V2

### ISSUE THREAD

**Issue by anthony-chaudhary** [diffgram/diffgram#821]
## Intents
**The intent is that it's always ease to move around the user provided point cloud.**

* The camera should match the scene (arbitrary user provided data).
Becuase as long as the camera orientation is aligned well with the scene, the controls including `WASD` etc make sense.
* Any helpers like grid etc, should be orientated in a meaningful way
* The system should somehow (be it user declared or automatic) have an idea of "up/down" as this will effect future validation things will depend on that.

## Implementation notes
The main challenge appears to be that the point cloud can be orientated in any arbitrary way.

In our current context the camera does not appear to align with the provided data, which makes certain controls not work as expected.

* Modifying anything, including the camera, can have side effects.
Perhaps this is because to determine position we are doing intersections with the camera.
For example this may be relevant
https://stackoverflow.com/questions/8272297/three-js-rotate-projection-so-that-the-y-axis-becomes-the-z-axis
But naively doing it effected annotation.

* Getting a clearer idea of what we can automatically determine, and what must be declared at some point by the user.

* Get a clearer idea on difference between expectations on coordinate systems vs orientation changes. For example some self driving cars use cases 

> The camera frame is placed in the center of the camera lens. The x-axis points down the lens barrel out of the lens. The z-axis points up. The y/z plane is parallel to the sensor plane. The coordinate system is right handed.

 https://github.com/waymo-research/waymo-open-dataset/issues/24#issuecomment-527956868
 

### So to sum it up it up, the next version may be something like:
1) Select coordinate system (e.g. Z Up or not)
2) Check assumptions on rest of system (helpers, annotation things etc) work as expected under new assumptions.
